ambry-tools/src/main/java/com.github.ambry/store/DumpData.java

```diff
@@ -138,12 +138,13 @@ public static void main(String args[]) {
               .defaultsTo("false")
               .ofType(String.class);
 
-      ArgumentAcceptingOptionSpec<Integer> bytesPerSecOpt =
-          parser.accepts("bytesPerSec", "Allowed bytes per sec for the purpose of throttling")
-              .withRequiredArg()
-              .describedAs("bytesPerSec")
-              .ofType(Integer.class)
-              .defaultsTo(0);
+      ArgumentAcceptingOptionSpec<Integer> bytesPerSecOpt = parser.accepts("bytesPerSec",
+          "Allowed bytes per sec for the purpose of throttling. Any value greater than 0 "
```

Comment:
Maybe we can put something like `... 0 to disable throttling`. And when a negative value is provided, simply throw `IllegalArgumentException`.